packages/global-styles/src/components/global-styles-variation-container/index.tsx

```diff
@@ -74,6 +79,9 @@ const GlobalStylesVariationContainer = ( {
 			<EditorStyles styles={ editorStyles ?? [] } />
 			{ containerResizeListener }
 			{ children }
+			{ fontFamilies && fontFamilies.length > 0 && (
+				<FontFamiliesLoader fontFamilies={ fontFamilies } onLoad={ onFontFamiliesLoad } />
```

Comment:
Why do we need to load the fonts here? I believe, every theme should include ALL the fonts referenced by the style variations, even if they are Jetpack Google fonts. So all fonts must have been printed by D132838-code.

Or could you point to a theme (+ variation) which won't work if we don't load it via Google Fonts here?